selenium-webdriver/http.d.ts

```diff
@@ -100,10 +100,11 @@ declare namespace http {
      */
     class Executor {
         /**
-         * @param {!HttpClient} client The client to use for sending requests to the
-         *     server.
+         * @param {!(HttpClient|IThenable<!HttpClient>)} client The client to use for sending
```

Comment:
On closer inspection, HttpRequest and HttpResponse are named wrong (though apparently that's been the case), I'll fix those